src/neon/transformers/passes/pybindwrapperpass.py

```diff
@@ -336,6 +336,12 @@ def flatten(self, container):
         else:
             yield container
 
+    @visit.on_type(ConcatOp)
+    def visit(self, op):
+        self.computation.set_op_rank(op)
+        self.computation.register_cpp_op(
+            op, self.computation.lookup_cpp_op(op.ops[-1]), set_name=False)
```

Comment:
Just a question. Where are we registering the cpp concat op? I guess it should happen here.